src/main/java/frc/robot/Constants.java

```diff
@@ -70,4 +79,9 @@
     public static final int GRAYHILL_ENCODER_DISTANCE_PER_PULSE = 1; //TODO: Replace this value with a correct one 
 
     public static final int XBOX_A = 1;
+
+    // Vision Subsystem
+    public static final double TOLERANCE_DEG = 5.;
+    public static final double STABALIZE_SECS = .5;
+    public static final double TIMEOUT_SECS = 3;
```

Comment:
These constants are not part of vision, they are part of the drivetrain system (used in TurnToHub). The names are not good because they are too generic. And "stabilize" is misspelled.